skimage/filters/thresholding.py

```diff
@@ -1022,3 +1024,92 @@ def apply_hysteresis_threshold(image, low, high):
     connected_to_high = sums > 0
     thresholded = connected_to_high[labels_low]
     return thresholded
+
+
+def threshold_multiotsu(image, classes=3, bins=255):
```

Comment:
```suggestion
def threshold_multiotsu(image, classes=3, bins=256):
```

The number of bins must be 256 as in the rest of the lib, otherwise, on int8 images, you will end up with 255/256 division.

Reply:
Is this a bug: https://github.com/scikit-image/scikit-image/blob/b573f9c092732b4beb3cc06d5fa37db5c7454c6f/skimage/io/_plugins/_histograms.pyx#L52